stickers.yml

```diff
@@ -19829,3 +19829,22 @@ e374df1d3ddbc818ee7f60972157c034:
     - blond
   nsfw: false
   original: true
+05480210c5eec8191c2dde3d248364c7:
+  key: ea1f7d6f3b94eab53cb3cc472fbeb78b502fc8e15082add959a6cfca7150e1b2
+  source: Apple Inc.
+  tags:
+    - T-Rex
+    - T Rex
+    - T
+    - Rex
+    - Dinosaur
+    - Dino
+    - Apple
+    - Memoji
+    - Emoji
+    - Iphone
+    - IPad
+    - Green
+    - Animal
+  nsfw: false
+  original: false
```

Comment:
Isn't this one also original?

Reply:
It is the stock Memoji version, imported from Apple keyboard, in iMessage. I converted the "artworks", but did not design them myself. Also they are available to any Apple user, who has access to Memoji on their device.

![image](https://user-images.githubusercontent.com/3185243/81349994-63763300-90c1-11ea-96da-6c5bc325e7a4.png)

btw ~ I am planning to add the rest of them, too

Reply:
Ho, great, I don't use iMessage so I had no idea. Thanks!